packages/unigraph-dev-explorer/src/components/ExplorerHome.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export default function ExplorerHome(){
+    // @ts-ignore
+    const [messages, setMessages]: [string[], Function] = React.useState(window.backendMessages);
+    // @ts-ignore
+    window.backendConnection.onmessage = (msg) => {
```

Comment:
just noticed this but should probably handle this with [`useEvent`](https://github.com/streamich/react-use/blob/master/docs/useEvent.md) from react-use